src/20_Components/amp-live-list.html

```diff
@@ -0,0 +1,146 @@
+      justify-content: space-around;
+    }
+    .content {
+      font-size: 1.5rem;
+    }
+    .flag {
+      padding-top: 8px;
+      padding-bottom: 8px;
+    }
+}
+  </style>
+</head>
+<body>
+
+  <div>
+    <h3>Live updates</h3>
+    <h4>Refresh takes 15 seconds.</h4>
+  </div>
+  <!-- #### amp-live-list with insert -->
+
+  <!-- 
+    Use `data-poll-interval` attribute to specify how frequently the `amp-live-list` component will poll new data. The minimum interval is 15 seconds. 
+    Use `data-max-items-per-page` to specify the maximum number displayed on the page.
+    For an extensive explanation of all the attributes, refer to [amp-live-list](https://github.com/ampproject/amphtml/blob/f2aed679a4c0611ea077985cbe0d1926c7d0a6f5/extensions/amp-live-list/amp-live-list.md).
+    
+    Use `on="tap:my-live-list.update"` to load the latest changes from the server. 
+    A button will appear when the AMP runtime discovers new document changes. Note that the button has to be defined as a direct child of `amp-live-list` .
+
+    A `amp-live-list` must contain a list of items below `<div items>`. Each item has a specific id and the `amp-live-list` will look for items with new id. 
+
+    <p class="card warning">
+    By clicking on the button "You have updates", new content is displayed on the demo column (the one on the right if the demo is accessed by desktop). The code column (the one in the center if the demo is accessed by desktop) does not update itself, but if you want to see all the items on the page, reload the page.
+    </p>
+
+    -->
+
+  <amp-live-list
+    layout="container"
+    data-poll-interval="15000"
+    data-max-items-per-page="5"
+    id="amp-live-list-insert-blog">
+    <button update on="tap:amp-live-list-insert-blog.update" class="button button-primary">You have updates</button>
+    <div items>
+      {{range .BlogItems}}
+          <div id="{{.Text}}" data-sort-time="{{.Timestamp}}">
+            <div class="card blog">
+                <amp-img src="{{.Image}}"
+                    layout="responsive" width="1280" height="853">
+                </amp-img>
+                <p>{{.Text}}</p>
+            </div>
+          </div>
+        {{end}}
+    </div>
+  </amp-live-list>
+
+  <!-- #### amp-live-list with update -->
+  <!-- 
+    When using `amp-live-list` with only updates on elements, the update button will not be shown.
+
+    Use `data-update-time` to specify when the update has been performed.
+
+    The following example will display a soccer match score whose content is changed every time the `amp-live-list` polls the page on the server.
+  -->
+    <amp-live-list
+    layout="container"
+    data-poll-interval="15000"
+    data-max-items-per-page="5"
+    id="amp-live-list-update">
+    <button id="btn" update on="tap:amp-live-list-update.update">You have updates</button>
+    <div items>
+        <div id="list-item-1" data-sort-time="1234567" data-update-time="{{.FootballScore.Timestamp}}">
```

Comment:
From [doc](https://github.com/ampproject/amphtml/blob/master/extensions/amp-live-list/amp-live-list.md)  data-sort-time is required. In this corner case it does not make a lot of sense, will follow up on that on a separate thread.